watertap/property_models/water_prop_pack.py

```diff
@@ -1059,9 +1060,17 @@ def rule_therm_cond_phase(b, p):
                 ) ** (
                     1 / 3
                 )
+            else:
+                raise FrozenPipes(
+                    f"Index '{p}' is not valid for indexed component 'fs.stream[0].therm_cond_phase"
+                )
             return (
                 b.therm_cond_phase[p]
-                == 10**log10_kw * 1e-3 * pyunits.W / pyunits.m / pyunits.K
+                == 10**log10_kw  # pylint: disable=possibly-used-before-assignment
```

Comment:
This seems like a failing of `pylint` if we really need this here -- this variable is either set or an exception is raised.